READMEs/quickstart.md

```diff
@@ -30,11 +30,10 @@ A video walkthrough should be available to assist in setting up. If not compress
 > ```
 
 > [!NOTE]
-> Make sure you socks is on with `vtk socks on` if you are not using local CMS preview.
+> Make sure you have a local VA CMS instance running if you are not using cached content.
```

Comment:
```suggestion
> Make sure you have a local VA CMS instance running.
```

Reply:
I would leave off the cached content part since we don't have instructions on how to set something like that up.